industry/companies.md

```diff
@@ -35,6 +41,12 @@ Raised pre-seed from Blue Ridge Foundation and $4M from General Catalyst, AIX Ve
 - [imagetranslate.com](https://www.imagetranslate.com/)
 - [Crunchbase - ImageTranslate](https://www.crunchbase.com/organization/imgtranslate)
 
+### Zhongke Fanyu Technology
+- [zkfy.com](http://www.zkfy.com/)
+- [Crunchbase - Zhongke Fanyu Technology](https://www.crunchbase.com/organization/zhongke-fanyu-technology)
+
+Incubated by the Chinese Academy of Sciences’ Institute of Automation
```

Comment:
If I had to bet, I'd bet this sentence will be machine-translated poorly. :-)

I'd make it:

> Incubated by the Institute of Automation at the Chinese Academy of Sciences

Or maybe just
> Supported by the Institute of Automation at the Chinese Academy of Sciences